2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,11 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
+  let twoPantry = [...pantry.fridgeContents, ...pantry.cupboardContents];
```

Comment:
Good usage of the spread operator. 
https://www.educative.io/answers/spread-operator-vs-arrayprototypeconcat-in-javascript